codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicheQuicChannel.java

```diff
@@ -1191,6 +1216,28 @@ private boolean connectionSendSimple() {
         return packetWasWritten;
     }
 
+    private boolean writePacket(DatagramPacket packet, int len) {
+        ChannelFuture future = parent().write(packet);
+        if (isSendWindowUsed(len)) {
+            // Nothing left in the window, continue later
+            future.addListener(continueSendingListener);
+            return true;
+        }
+        return false;
+    }
+
+    private static boolean isSendWindowUsed(int len) {
+        return len < Quic.MAX_DATAGRAM_SIZE;
+    }
+
+    private static int calculateSendBufferLength(long connAddr) {
+        int len = Math.min(Quic.MAX_DATAGRAM_SIZE, Quiche.quiche_conn_send_quantum(connAddr));
+        if (len <= 0) {
+            return 8;
```

Comment:
hi there. I strolled through your commit and wondered about the '8'. Then I wondered about the whole condition.
Shouldn't be the minimum of a define and something variable not the define in the end?
But anyway, '8' sounded very small. So where does this magic number come from?